javascript/javascript1/week3/whenWillWeBeThere.js

```diff
@@ -0,0 +1,18 @@
+// WHEN WILL WE BE THERE
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+  
+const travelTimeInMinutes = travelInformation.destinationDistance * 60 / travelInformation.speed;
+  
+function convert(num) {
```

Comment:
I would suggest to make a function 
```suggestion
function getTotalTripTime(information) {
```
which will take information as a parameter and return the calculations (as a sentence) 
In that case `const travelTimeInMinutes = travelInformation.destinationDistance * 60 / travelInformation.speed;` should be inside the function.